playbooks/roles/airship-deploy-ucp/defaults/main.yml

```diff
@@ -2,3 +2,5 @@
 airship_pegleg_image: "{{ suse_airship_registry_location }}/airshipit/pegleg:{{ suse_airship_components_image_tag['pegleg'] }}"
 run_tests: true
 test_timeout: 2700
+neutron_external_interface: public
```

Comment:
This interface does not exist by default. So this will just break when deploying the ovs-agent.

Reply:
changed default to empty.